Imagehandler.py

```diff
@@ -48,36 +49,38 @@ def WritingImage(self, image, path, imageName):
             cv.imwrite(path + imageName, image)
             cv.imshow(imageName, image)
             cv.waitKey(0)
+        cv.destroyAllWindows()
```

Comment:
Alignment issue

Reply:
If we are not using this(line no 52), then the program crashed after showing the image.
![screenshot from 2017-09-04 17-59-44](https://user-images.githubusercontent.com/22221731/30026597-3d72d1fe-919b-11e7-834a-81136ba3f71c.png)

Reply:
is indentation correct?